applications/pelion_client/README.rst

```diff
@@ -0,0 +1,522 @@
++===============================+===========================+================================================================================+
+| Network back end disconnected | **LED1** blinking         | NET_STATE_DISABLED or NET_STATE_DISCONNECTED                                   |
++-------------------------------+---------------------------+--------------------------------------------------------------------------------+
+| Pelion connection search      | **LED2** blinking         | PELION_STATE_DISABLED or PELION_STATE_INITIALIZED or PELION_STATE_UNREGISTERED |
++-------------------------------+---------------------------+--------------------------------------------------------------------------------+
+| Network back end connected    | **LED1** solid on         | NET_STATE_CONNECTED                                                            |
++-------------------------------+---------------------------+--------------------------------------------------------------------------------+
+| Pelion connection established | **LED2** solid on         | PELION_STATE_REGISTERED                                                        |
++-------------------------------+---------------------------+--------------------------------------------------------------------------------+
+
+Requirements
+************
+
+The application supports the following development kits:
+
+.. table-from-rows:: /includes/sample_board_rows.txt
+   :header: heading
+   :rows: nrf9160dk_nrf9160ns, nrf5340dk_nrf5340_cpuapp, nrf52840dk_nrf52840
+
+.. include:: /includes/spm.txt
+
+Pelion Device Management requirements
+=====================================
+
+You need a developer account on the `Pelion Device Management Portal`_.
+
+Configuration
+*************
+
+The Pelion library is enabled with :option:`CONFIG_PELION_CLIENT` configuration option.
+
+For the library to work, you must enable and properly configure the Mbed TLS library.
+To connect to the Device Management, the device must be provisioned with valid credentials.
+For the firmware update procedure to work, you must provision the device with valid update resources.
+
+Mbed TLS
+========
+
+The Pelion library is depending on a properly configured Mbed TLS library.
+To simplify the development within the |NCS|, a predefined set of Mbed TLS configuration options was prepared.
+Set compatible with :ref:`nrfxlib:nrf_security` can be enabled using :option:`CONFIG_PELION_NRF_SECURITY` configuration option.
+
+Pelion credentials
+==================
+
+To be able to connect to the Device Management, you need to complete the following configuration:
+
+1. Generate and download a developer certificate file from the `Pelion Device Management Portal`_.
+   This file contains keys used for securing connection with the Pelion network.
+#. Copy the downloaded file to :file:`applications/nrf_desktop/configuration/common/`, replacing the default :file:`mbed_cloud_dev_credentials.c` file.
```

Comment:
File location should be `applications/pelion_client/configuration/common/`

Reply:
Fixed